Assets/Runtime/UIFrame.cs

```diff
@@ -304,6 +340,22 @@ public class UIFrame : MonoBehaviour
             type = null;
             return false;
         }
+        
+        /// <summary>
+        /// Checks if a given screen id is opened either on the Window or Panel layers
+        /// </summary>
+        /// <param name="screenId">The Id to check.</param>
+        public bool IsScreenOpened(string screenId) {
+            if (windowLayer.IsScreenRegistered(screenId)) {
```

Comment:
if we got more than 1 layer, mb need to iterate over them somehow? =)

Reply:
@DevFrommars not sure I get comment, pls clarify what can be changed/improved
but just in case: uiframework support 2 kind of layers now - windowLayer  & panelLayer and they have slightly different API at that moment

Reply:
understood, resolving